src/index.scss

```diff
@@ -1,3 +1,9 @@
+@import '../node_modules/carbon-components/scss/globals/scss/styles.scss';
+
+$feature-flags: (
```

Comment:
$feature-flags should go above styles import. It works for use because you use 10.3 version of "carbon-components". With 10.2 it won't work.

Reply:
Just FYI 😄

Reply:
Good to know, thanks!